src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/ReplyToMessage.vue

```diff
@@ -45,26 +65,109 @@
 </template>
 
 <script lang="ts">
-import { defineComponent } from "vue";
+import type { AxiosProgressEvent } from "axios";
+import { v4 as uuidv4 } from "uuid";
+import { defineComponent, ref } from "vue";
 import { useRouter } from "vue-router";
 
+import { uploadFile } from "@/api/file";
 import { sendMessage } from "@/api/message";
 import ConfirmationDialog from "@/components/ConfirmationDialog.vue";
 import PageContainer from "@/components/PageContainer.vue";
+import UploadFileItem from "@/components/UploadFileItem.vue";
 import { useAlertStore } from "@/store/alert";
 import { useLoadingStore } from "@/store/loading";
 import { useMessageStore } from "@/store/message";
 import * as Rules from "@/utils/formRules";
 
 export default defineComponent({
   name: "ReplyToMessage",
-  components: { PageContainer, ConfirmationDialog },
+  components: { PageContainer, ConfirmationDialog, UploadFileItem },
   setup() {
     const messageStore = useMessageStore();
     const loadingStore = useLoadingStore();
     const alertStore = useAlertStore();
     const router = useRouter();
-    return { messageStore, loadingStore, alertStore, router };
+    const fileInput = ref<HTMLInputElement | null>(null);
+    const selectedFiles = ref<{ file: File; progress: number }[]>([]);
+    const formatFileTags = (file: File): string => {
+      const fileName = file.name;
+      const fileSize = (file.size / 1024).toFixed(2) + " KB"; // Convert size to KB
```

Comment:
I think we need to also take into account files that are MB in size. 

this function should handle it? or you can come up with one too 
https://github.com/bcgov/EDUC-STUDENT-ADMIN/blob/64fba78d4d08835c1975cb6cfa2f7fa41b388eac/frontend/src/utils/file.js#L7